integreat_compass/cms/templates/index.html

```diff
@@ -79,6 +80,46 @@ <h3 class="text-gray-500 mt-10">{% translate "Categories" %}</h3>
                             {% endfor %}
                         </ul>
                     </div>
+                    <div class="mt-4">
+                        <div>
+                            <label for="filter-address">{% translate "Location" %}</label>
+                            <input type="text"
+                                   name="filter-address"
+                                   placeholder="{% translate "Address" %}"
+                                   id="filter-address">
+                        </div>
+                        <div class="hidden">
+                            <input type="number"
+                                   class="filter-element"
+                                   name="lat"
+                                   step="1e-7"
+                                   id="id_location-lat">
+                            <input type="number"
+                                   class="filter-element"
+                                   name="long"
+                                   step="1e-7"
+                                   id="id_location-long">
+                        </div>
+                        <ul class="mt-2 items-center w-full border border-gray-500 rounded flex shadow-inner">
+                            {% for distance in offer_distance_steps %}
```

Comment:
This section also overflows for me on smaller laptop / tablet screens. 
![image](https://github.com/digitalfabrik/integreat-compass/assets/72705147/28bcfea4-1b49-4641-b771-5c10f8c03767)

This gets even worse if we agree to add the overflow-y-scroll as mentioned above, because I rather have an overflowing element than have it vanish on smaller screens. Unfortunately I wasn't able to come up with a quick yet efficient solution for it.